noise:
  gyroscope_std: 0.01
  accelerometer_std: 0.1
  gyroscope_bias_std: 0.0001
  accelerometer_bias_std: 0.001
  landmark_std: 0.1
  contact_std: 0.01

